unit_testing.sh

```diff
@@ -1,5 +1,4 @@
 #/bin/sh
 source $HOME/build/neuropoly/spinalcordtoolbox/python/bin/activate
 export PYTHONPATH=$PYTHONPATH:$SCT_DIR/scripts
-pytest -v --cov=spinalcordtoolbox.deepseg_gm unit_testing/test_deepseg_gm.py
-pytest -v --cov=spinalcordtoolbox.metadata unit_testing/test_metadata.py
+pytest -v --cov-append --cov=spinalcordtoolbox --cov=scripts unit_testing/*
```

Comment:
It's ok now, but eventually when we have all tests, it should be looking into root dir.